src/js/engagement_view/src/components/reusableComponents/collapsibleDrawer.tsx

```diff
@@ -107,40 +116,56 @@ export default function CollapsibleNavDrawer() {
                 </DrawerHeader>
                 <Divider />
                 <List className={classes.drawer}>
-                    <ListItem button key="Analyzers">
+                    <ListItemButton
+                        key="Analyzers"
+                        selected={selectedIndex === 0}
+                        onClick={(event) => handleListItemClick(event, 0)}
+                    >
```

Comment:
I won't stop the PR on this, but this section definitely has a code smell. Consider doing something like the following:
```
interface ListItem {
  label: String,
  link: String,
}

const items: ListItem[] = [
  { label: "Analyzers", link: "/analyzers" },
  { label: "Engagements", link: "/engagements" },
];

items.forEach((value, index) => {
	// render each item as a ListItemButton
});

```

Reply:
Used a new feature to set active links in upgrade to react router dependency.